src/main/scala/tech/sourced/gemini/ConnectedComponents.scala

```diff
@@ -0,0 +1,125 @@
+package tech.sourced.gemini
+
+import java.nio.ByteBuffer
+
+import com.datastax.driver.core.{Session, SimpleStatement}
+import org.slf4j.{Logger => Slf4jLogger}
+import scala.collection.JavaConverters._
+import scala.collection.mutable.ListBuffer
+
+case class FileHash(sha1: String, value: ByteBuffer)
+
+class DBConnectedComponents(
+                              log: Slf4jLogger,
+                              conn: Session,
+                              keyspace: String = Gemini.defautKeyspace) extends ConnectedComponents(log) {
```

Comment:
Here and below, for the class declarations, its a good idea to check that the style guide is followed http://docs.scala-lang.org/style/declarations.html#classes 

In this case, the preferred way would be:
```scala
class DBConnectedComponents(
    log: Slf4jLogger,
    conn: Session,
    keyspace: String = Gemini.defautKeyspace) 
  extends ConnectedComponents(log) {
}
```

Reply:
I moved extends to a new line. But indent is still large, IntelliJ IDEA does it.